src/plugins/console/public/application/components/split_panel/split_panel.test.tsx

```diff
@@ -11,7 +11,8 @@ import { mount } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import { spy } from 'sinon';
 
-import { PanelsContainer, Panel } from './index';
+import { Panel } from '../../containers/split_panel/panel';
```

Comment:
Please, can you move tests of `Panel` and `PanelsContainer` to the folder of those components?